apps/spruce/src/pages/adminSettings/tabs/GeneralTab/BackgroundProcessingTab/schemaFields.ts

```diff
@@ -267,13 +267,20 @@ export const triggers = (distros: string[]) => ({
     generateTaskDistro: {
       type: "string" as const,
       title: "Distro for Generated Tasks",
-      oneOf: [
-        ...distros.map((d) => ({
-          type: "string" as const,
-          title: d,
-          enum: [d],
-        })),
-      ],
+      oneOf:
```

Comment:
without this i found that the page was crashing on staging, i'm not sure what a better approach is... seems like it takes a second for the distros to load in

Reply:
looks good to me